README.md

```diff
@@ -137,7 +137,7 @@ To setup your PostgreSQL configuration with a specific data directory. If you ha
 postgresql_server_conf 'My PostgreSQL Config' do
   version '9.5'
   data_directory '/data/postgresql/9.5/main'
-  notification :reload
+  notifies :reload, 'service['postgresql']
```

Comment:
I actually, sadly, think this is still wrong. Thanks for the PR though. This service name doesn't always exist because distros like having their own names.